### ISSUE TITLE
Scrollbar disappear while scrolling all the way down

### ISSUE THREAD

**Issue by gork7777** [AdguardTeam/AdGuardVPNExtension#87]
### Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard VPN Browser Extension version

Beta 1.4.14

### Environment

```markdown
- OS: Windows 10
- Browser: Chrome v111.0.5563.111 - Firefox v111.0.1
```

### Issue Details

The scrollbar disappear while scrolling in the "Locations" section. And the UI is a bit distorted for Firefox, scrolling the rightmost scrollbar creates a dark blue rectangle in the middle.

### Expected Behavior

The scrollbar should be visible when scrolling. And the UI for Firefox should be improved, there shouldn't be a dark blue rectangle in the middle.

### Screenshots

<details><summary>Chrome</summary>

https://user-images.githubusercontent.com/35523759/228234913-9916769e-39e5-40da-a5a5-4084139fe58a.mp4
</details>

<details><summary>Firefox</summary>

https://user-images.githubusercontent.com/35523759/228235018-3885596d-0600-47bc-ac7f-3020f5dbf9e5.mp4
</details>

<details><summary>Firefox UI</summary>

![uii](https://user-images.githubusercontent.com/35523759/228236946-2a483198-e911-47ef-be8a-bcbe10ccf885.png)
</details>

**Comment by mozartinio** [comment]
@gork7777 
Hello!
Thank you! We are working on it

**Comment by gork7777** [comment]
@mozartinio Looks like issue is fixed in 1.4.20.
